hedera-node/docs/design/services/smart-contract-service/system-contract-abi-guidelines.md

```diff
@@ -0,0 +1,132 @@
+# System Contract API Design Guidelines
+
+(**A start at answering [#14244](https://github.com/hashgraph/hedera-services/issues/14244)**)
+
+This document contains design guidelines for system contract ABIs.
+
+## Motivation
+
+- We’ve learned over time that the APIs we build for system contracts have an impact on
+    - usability,
+    - maintainability,
+    - security.
+- API design shouldn’t be *ad hoc* - there should be consistency that supports the goals above
+  (and avoids mistakes).
+- System contracts are *contract* interfaces that  expose HAPI logic in the equivalent manner of an 
+  EVM precompile. System contracts preserve the behaviour of an EVM smart contract but capitalize of
+  the power of native services. System contracts are intended to be used by smart contract 
+  *developers* in accordance with solidity development best practices: Therefore the APIs should be 
+  designed with those developer’s common expectations and practices in mind.
+
+## Related
+
+- The question of how to _version_ system contracts.
+  - Which is being worked on in a different issue, and when there's a PR for it will be referenced 
+    here.
+
+## Guidelines
+
+*(An unorganized list at the moment, and needing comments, additions, and review!)*
+
```

Comment:
Should there be a comment that these guidelines have not been adhere to strictly in the past so that the user is not confused by the current state of the world.  Rather these guidelines are for going forward.